### PR TITLE
added actor id to show condition

### PR DESCRIPTION
**Pull request by xnetsrak**
## Description

added actor id to show condition
## References

- #0000

### CODE REVIEW

File: libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html [hunk 1]
```diff
@@ -36,14 +36,24 @@
     <!-- sender gln -->
     <ng-container matColumnDef="sender">
       <th mat-header-cell *matHeaderCellDef>{{ transloco("sender") }}</th>
-      <td mat-cell *matCellDef="let element">{{ element.senderGln }}</td>
+      <td mat-cell *matCellDef="let element">
+        <container-element [ngSwitch]="element.httpData">
+          <label *ngSwitchCase="'request'" type="info">{{
```
Comment (dzhavat) [inline libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html:41]: suggestion: The `label` element is usually used in relation to other elements like `input`, `textarea`, etc. I can see in the rest of the file that the `watt-badge` component was used for a similar case. Could it be that `watt-badge` should be used here too?

File: libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html [hunk 2]
```diff
@@ -36,14 +36,24 @@
...
+            element.senderGln
+          }}</label>
+          <label *ngSwitchCase="'response'" type="success">{{
+            element.receiverGln
+          }}</label>
+          <label *ngSwitchDefault type="info">{{ "??" }}</label>
```
Comment (dzhavat) [inline libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html:47]: nitpick: the interpolation (`{{ .. }}`) is not necessary when we want to output a simple text string.

I.e. `<element>??</element>`

File: libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html (hunk 1 above)
Comment (xnetsrak) [reply libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html:41]: no badge here it should only be text

Comment (dzhavat) [reply libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html:41]: Then unless you need specific styling, you can replace `label` with `ng-container` and remove the `type` property. `ng-container` is a generic element in Angular that will not render the element in the HTML DOM tree.

**Review by dzhavat — APPROVED** [review]

**Pull request merged**